client/app/cloud/project/compute/loadbalancer/configure/compute-loadbalancer-configure.less

```diff
@@ -0,0 +1,31 @@
+#compute-loadbalancer-configure {
```

Comment:
You shouldn't have to do that.  ovh-ui-kit should suffice in most case

Reply:
done, but the alignment between oui-list and oui-button is bad, what I must do ?